src/plugins/plugin.tooltip.js

```diff
@@ -1073,9 +1077,9 @@ export class Tooltip extends Element {
 	 * @returns {boolean} True if the position has changed
 	 */
   _positionChanged(active, e) {
-    const me = this;
-    const position = positioners[me.options.position].call(me, active, e);
-    return me.caretX !== position.x || me.caretY !== position.y;
+    const {caretX, caretY, options} = this;
+    const position = positioners[options.position].call(this, active, e);
+    return caretX !== valueOrDefault(position.x, caretX) || caretY !== valueOrDefault(position.y, caretY);
```

Comment:
Not sure if it would be more efficient this way:
```suggestion
    return position !== false && (caretX !== position.x || caretY !== position.y);
```

Reply:
That's simpler. I'll go with it